### PR TITLE
Blitz: docstring & indentation fixes

### PR DESCRIPTION
**Pull request by simleo**
This PR fixes a few docstrings in the blitz code. It also reindents `blitz/src/ome/formats/model/*.java`, where a mix of different conventions (most notably tabs and spaces) makes the code hard to read.

**TEST:** check that it does not break anything

### CODE REVIEW

File: components/blitz/src/ome/formats/model/PixelsProcessor.java [hunk 1]
```diff
@@ -44,10 +44,8 @@
 
 /**
  * Processes the pixels sets of an IObjectContainerStore and ensures
- * that the physical pixel dimensions are updated if they were specified by
- * the user. If Image containers are present, Image.acquisitionDate is filled
- * out and that the Image name and description match that which was specified
```
Comment (joshmoore) [inline components/blitz/src/ome/formats/model/PixelsProcessor.java]: @simleo : then perhaps let's also remove the commented code and file a card to come back to the issue on devx?

Comment (simleo) [inline components/blitz/src/ome/formats/model/PixelsProcessor.java]: Remove the commented code in *this* PR?

Comment (joshmoore) [inline components/blitz/src/ome/formats/model/PixelsProcessor.java]: Since they go together, yes. Or revert and leave it for someone else.

**Comment by ximenesuk** [comment]
The further changes look fine to me. And nothing broke.

**Comment by joshmoore** [comment]
Created https://trello.com/c/PggYSt2Y/607-remove-commented-out-import-code / merging.

**Pull request merged**